### ISSUE TITLE
test

### ISSUE THREAD

**Issue by wenhangyu** [liuh127/NTIRE-2021-Dehazing-DWGAN#5]
Thanks for your reply, now I have another question, I have downloaded the dataset on the official website, but I found that only the training set has haze images and GT, and there is no GT in the validation set and test set. If the test image does not have GT, how can I get the PSNR result?
